t/03-testapi.t

```diff
@@ -1038,4 +1039,24 @@ subtest 'mouse click' => sub {
     is $cmds->[0]{button}, 'left', 'mouse_tclick called with default button' or diag explain $cmds;
 };
 
+is_deeply testapi::_handle_found_needle(undef, undef, undef), undef, 'handle_found_needle returns no found needle by default';
+$bmwqemu::vars{CASEDIR} = 'foo';
+is get_test_data('foo'), undef, 'get_test_data can be called';
+lives_ok { become_root } 'become_root can be called';
+like(exception { ensure_installed }, qr/implement.*for your distri/, 'ensure_installed can be called');
+lives_ok { hold_key('ret') } 'hold_key can be called';
+lives_ok { release_key('ret') } 'release_key can be called';
+lives_ok { reset_consoles } 'reset_consoles can be called';
+lives_ok { power('on') } 'power can be called';
+lives_ok { save_memory_dump } 'save_memory_dump can be called';
+like(exception { save_storage_drives }, qr/should be called.*post_fail_hook/, 'save_storage_drives should be called special');
+lives_ok { freeze_vm } 'freeze_vm can be called';
+lives_ok { resume_vm } 'resume_vm can be called';
```

Comment:
Having tests that only check if a function does not die might increase test coverage, but we don't actually test that the functions are doing what we expect. Is that still planned, or do you think we don't need it here for some reason?

Reply:
Well, I only added tests to cover the call signatures. I think having better functional tests would be good but that's not related to introducing signatures, right?